firebase-vscode/src/cli.ts

```diff
@@ -172,6 +177,32 @@ export async function initHosting(options: { spa: boolean; public: string }) {
   await initAction("hosting", commandOptions);
 }
 
+export async function emulatorsStart(firebaseJson: FirebaseConfig, emulatorUiSelections: EmulatorUiSelections) {
+  const commandOptions = await getCommandOptions(undefined, {
```

Comment:
Seems like this  should take firebaseJson as the first arg instead of undefined?

Reply:
This is a remnant of a previous build - removed this for now since the workflow.ts caches the active firebaseJSON there's no need to pass it around.